### PR TITLE
Update to docs

### PR DESCRIPTION
**Pull request by ofermend**
# Description

Minor update to docs, adding an example for how to use VectaraIndex to connect to existing corpus without adding new documents. Just to make it super clear.

- [X] This change requires a documentation update

# How Has This Been Tested?

- [X] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Review by hatianzhang — APPROVED** [review]
LGTM!

File: llama_index/indices/managed/vectara/retriever.py [hunk 1]
```diff
@@ -33,17 +33,15 @@ class VectaraRetriever(BaseRetriever):
...
-            * mmr_k: number of results to fetch for MMR, defaults to 50
-            * mmr_diversity_bias: number between 0 and 1 that determines the degree
-                of diversity among the results with 0 corresponding
-                to minimum diversity and 1 to maximum diversity.
-                Defaults to 0.3.
-            * summary_enabled: whether to generate summaries or not. Defaults to False.
-            * summary_response_lang: language to use for summary generation.
-            * summary_num_results: number of results to use for summary generation.
-            * summary_prompt_name: name of the prompt to use for summary generation.
+        mmr_k: number of results to fetch for MMR, defaults to 50
```
Comment (hatianzhang) [inline llama_index/indices/managed/vectara/retriever.py:36]: it seems those parameters are not passed for init but as kwargs. is it what you want?

Comment (ofermend) [reply llama_index/indices/managed/vectara/retriever.py:36]: yes good point. The number of parameters keeps growing, but I see your point - if it's listed in the docstring might as well add it to the actual thing.
Let me update with this change.

Comment (ofermend) [reply llama_index/indices/managed/vectara/retriever.py:36]: updated!

**Comment by hatianzhang** [comment]
awesome LGTM

**Pull request merged**